CONTRIBUTING.md

```diff
@@ -225,6 +225,36 @@ discussed extensively on the mailing list and IRC, be accompanied by a widely
 discussed BIP and have a generally widely perceived technical consensus of being
 a worthwhile change based on the judgement of the maintainers.
 
+### Finding Reviewers
+
+As most reviewers are themselves developers with their own projects, the review
+process can be quite lengthy, and some amount of patience is required. If you find
+that you've been waiting for a pull request to be given attention for several
+months, there may be a number of reasons for this, some of which you can do something
+about:
+
+  - It may be because of a feature freeze due to an upcoming release. During this time,
+    only bug reports are taken into consideration. If your pull request is a new feature,
+    it will be disregarded until the release is over. Wait for release.
+  - It may be because the changes you are suggesting do not appeal to people. Rather than
+    nits and critique, which require effort and means they care enough to spend time on your
+    contribution, thundering silence is a good sign of widespread (mild) dislike of a given change
+    (because people don't assume *others* won't actually like the proposal). Don't take
+    that personally, though! Instead, take another critical look at what you are suggesting
+    and see if it: changes too much, is too broad, doesn't adhere to the [developer notes](doc/developer-notes.md),
+    is dangerous or insecure, is messily written, etc. Identify and address any of the
+    issues you find. Then ask e.g. on IRC if someone could give their opinion on the concept
+    itself.
+  - It may be because your code is too complex for all but a few people. And those people
+    may not have realized your pull request even exists. A great way to find people who
+    are qualified and care about the code you are touching is the [Git Blame feature](https://help.github.com/articles/tracing-changes-in-a-file/). Simply
```

Comment:
Good suggestion.

It looks like github also provides reviewer suggestions based on who touched the code before (see e.g. https://github.com/bitcoin/bitcoin/pull/10202, or any pull that hasn't got any reviewers yet)